### PR TITLE
Feature/573 Button Styling

### PR DESCRIPTION
**Pull request by ManuelMoeri**

### CODE REVIEW

**Review by MarcEgliP — DISMISSED** [review]

File: frontend/src/app/action-plan/action-plan.component.scss [hunk 1]
```diff
@@ -61,5 +61,5 @@
 }
 
 .add-action-text {
-  margin-bottom: 5px;
```
Comment (kcinay055679) [inline frontend/src/app/action-plan/action-plan.component.scss:64]: maybe combine with other class?

File: frontend/src/app/objective/objective.component.scss [hunk 2]
```diff
@@ -46,5 +46,5 @@
 }
 
 .add-keyresult-text {
-  margin-bottom: 5px;
```
Comment (kcinay055679) [inline frontend/src/app/objective/objective.component.scss:49]: maybe combine with other class?

File: frontend/src/style/custom_angular.components.scss [hunk 3]
```diff
@@ -12,6 +13,15 @@
   }
 }
 
+.mdc-button--outlined {
+  padding: 0 16px 0 16px !important;
+  margin-left: 8px !important;
+}
+
+.mat-mdc-button {
+  margin-left: 8px !important;
```
Comment (kcinay055679) [inline frontend/src/style/custom_angular.components.scss:22]: there is already a class with this selector on line 3

File: frontend/src/style/custom_angular.components.scss [hunk 4]
```diff
@@ -25,16 +35,12 @@
   width: 100% !important;
 }
 
-mat-dialog-content,
-mat-dialog-actions {
-  padding: 0 1.62rem 0 1.62rem !important;
```
Comment (kcinay055679) [inline frontend/src/style/custom_angular.components.scss:30]: why is this required?

File: frontend/src/style/styles.scss [hunk 5]
```diff
@@ -262,3 +262,7 @@ mat-form-field.quarter-filter .mat-mdc-text-field-wrapper {
 .sub-title-keyresult {
   font-size: 0.9rem !important;
 }
+
+.add-cross-button {
+  margin-bottom: 1px;
```
Comment (kcinay055679) [inline frontend/src/style/styles.scss:267]: maybe combine with other class?

**Review by kcinay055679 — CHANGES_REQUESTED** [review]
